### PR TITLE
Added support for Prometheus.

### PR DESCRIPTION
**Pull request by kimmelserj**

### CODE REVIEW

File: src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -37,6 +37,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('source')->defaultNull()->end()
                             ->scalarNode('username')->defaultNull()->end()
                             ->scalarNode('influxdb_client')->defaultNull()->end()
+                            ->scalarNode('prometheus_collector_registry')->defaultNull()->end()
+                            ->scalarNode('prometheus_namespace')->defaultValue('')->end()
```
Comment (lyrixx) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php]: What about calling this one `namespace` ?

File: src/Beberlei/Metrics/Collector/Prometheus.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    {
+        $this->measure($variable, $time);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function flush()
+    {
+    }
```
Comment (lyrixx) [inline src/Beberlei/Metrics/Collector/Prometheus.php]: This method is here to be able to send all "events" in an async way. 
With the current implementations, everything is sent in real time.
Could you update the code to mimic others implementations ?

File: src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -37,6 +37,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('source')->defaultNull()->end()
                             ->scalarNode('username')->defaultNull()->end()
                             ->scalarNode('influxdb_client')->defaultNull()->end()
+                            ->scalarNode('prometheus_collector_registry')->defaultNull()->end()
```
Comment (stof) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php]: you need to add a validation rule making sure that this one is not empty for ``prometheus`` collectors (see below how it is done for librato validation)

Comment (stof) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php]: I suggest using ``->info()`` here to explain that this is a service id

File: src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (kimmelserj) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php]: Ok. I will rename it to `namespace`.

File: src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php (hunk 3 above)
Comment (kimmelserj) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php]: Ok. I will add validation.

Comment (kimmelserj) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php]: Ok. I will add `->info()` for node `prometheus_collector_registry`.

**Comment by kimmelserj** [comment]
Done.

**Comment by kimmelserj** [comment]
@lyrixx, what else is required to accept this pull request?

**Comment by lyrixx** [comment]
Thanks @kimmelserj for working on this feature, this is much appreciated.

**Comment by lyrixx** [comment]
@kimmelserj I just release v2.3.0

**Pull request closed without merging**